src/test/java/domain/piece/KnightTest.java

```diff
@@ -0,0 +1,39 @@
+package domain.piece;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import domain.position.Position;
+import domain.position.Positions;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class KnightTest {
+    @DisplayName("나이트는 대각선, 행, 열을 제외한 가장 가까운 칸으로 이동할 수 있다.")
+    @Test
+    void knightMovable() {
+        //given
+        final Knight knight = new Knight(Team.BLACK);
+        Position source = Positions.from("D4");
+        final List<Position> destinations = Positions.of("E6", "C6", "B5", "F5", "B3", "F3", "C2", "E2");
+
+        //when
+
+        //then
```

Comment:
저는 이런경우 
```suggestion
        //given
        final Knight knight = new Knight(Team.BLACK);
        Position source = Positions.from("D4");
        final List<Position> destinations = Positions.of("E6", "C6", "B5", "F5", "B3", "F3", "C2", "E2");
        //when
        //then
```
주로 위와 같이 사용해요 :)

Reply:
보기 쉽게 바꿔봤습니다.